### ISSUE TITLE
Dependencies conflict with prometheus_client

### ISSUE THREAD

**Issue by alexdel2399** [perdy/starlette-prometheus#44]
Hello!
I've tried to use starlette_prometheus with prometheus-client==0.16.0, but there is a Dependencies conflict

```

ERROR: Cannot install -r requirements.txt (line 20) and prometheus-client==0.16.0 because these package versions have conflicting dependencies.

 

The conflict is caused by:
    The user requested prometheus-client==0.16.0
    starlette-prometheus 0.9.0 depends on prometheus_client<0.13 and >=0.12

 

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

```

Are there any reasons to install starlette_prometheus only with prometheus-client==0.12.0?

**Comment by sumankn314e** [comment]
@perdy I'm also facing the same issue can we get this fixed?

**Comment by perdy** [comment]
Thanks for raising this. It should be fixed with version 0.10
